### ISSUE TITLE
[MAS1.4.3] [Visual Requirements-Endpoint] Luminosity Ratio is less than 4.5:1 “Learn more and azure bot service configuration “links.

### ISSUE THREAD

**Issue by Kumar2608** [microsoft/BotFramework-Emulator#1794]
**Actual Result:**
Luminosity Ratio is less than 4.5:1 “Learn more and azure bot service configuration “links present in the Add an endpoint for your bot dialog 

**Expected Result:**
Luminosity Ratio should be greater than or equal to 4.5:1 for “Learn more and azure bot service configuration “links

**Repro Steps:**
1.Open the Bot Framework V4 Emulator.
2.Welcome tab screen gets opened
3.Navigate to the Bot explorer and select it 
4.Navigate to the End point symbol control and select it 
5.Dialog gets opened 
6.Open 'Colour Contrast Analyser' application.
7.Verify whether Luminosity Ratio is less than 4.5:1 for “Learn more and azure bot service configuration “links or not 

**User Impact:**
Users with low vision or colour blindness needs that difference between contrast of link is always high.

**Test Environment:**
OS Build: Version 1903(OS Build 18362.239)
Application: Bot Framework V4 Emulator
Tool: Colour contrast analyser

**MAS Reference Link:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}
![MAS1 4 3_End point symbol](https://user-images.githubusercontent.com/53518994/64008399-aac02300-cb33-11e9-8a60-0fe871d4c17e.png)

**Comment by tonyanziano** [comment]
This should be fixed by #1828 , but needs to be verified.
